### ISSUE TITLE
pwg:7-0484

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/csl-orig#461]
date:	01/10/2021 23:47:43
dict:	pwg
Lnum:	7-0484
hw:	saMsAra
old:	https://www.sanskrit-lexicon.uni-koeln.de/scans/PWGScan/2020/web/webtc/indexcaller.php LB A click on "printed book page" leads to the first page of the dictionary, not the requested page.
new:	
comm:	faulty link? LB Keep up the good work.

**Comment by funderburkjim** [comment]
This is a bug, no doubt introduced when working on API servepdf code.

It seems to be just with PWG.

Currently investigating.

**Comment by gasyoun** [comment]
> no doubt introduced when working on API servepdf code.

**Comment by funderburkjim** [comment]
Problem solved.

The problem occurred for both Grassman and PWG, because these two dictionaries have 2 page-numbers
on one image file.   When the page number is even (only in GRA and PWG),  the page number is
adjusted down by 1 to get the image file name.  This adjustment is done only for PWG or GRA,
and requires an accurate value for the dictionary code.

But when I reorganized the code for the api0 version,  I made an error in passing that 
dictionary code properly.  

Two lines were added in servepdfClass.php module of csl-apidev to correct the problem.
